core/utils/config/validate.go

```diff
@@ -33,7 +33,9 @@ func validate(v reflect.Value, checkInterface bool) (err error) {
 	if checkInterface {
 		i := v.Interface()
 		if vc, ok := i.(Validated); ok {
-			err = multierr.Append(err, vc.ValidateConfig())
+			for _, e := range utils.UnwrapError(vc.ValidateConfig()) {
+				err = multierr.Append(err, e)
+			}
 		} else if v.CanAddr() {
 			i = v.Addr().Interface()
 			if vc, ok := i.(Validated); ok {
```

Comment:
I think this one needs to be aligned with the previous case. Maybe we don't have coverage for it?

Reply:
yeah I don't think there's test coverage on it since adding/removing it didn't do anything

if the same issue comes up later on, then we at least know the solution for now - and it really only affects pretty-printing things